web/client/translations/data.de-DE

```diff
@@ -54,6 +54,41 @@
             "tooltipDeactivate": "Beenden Sie den 3D-Modus",
             "tooltipActivate": "Geben Sie den 3D-Modus ein"
         },
+        "cookie":{
+            "info": "This website uses cookies to improve your experience. We'll assume you're ok with this, but you can opt-out if you wish.",
+            "moreDetailsButton": "More Details",
+            "leave": "Leave",
+            "accept": "Accept",
+            "moreDetails": {
```

Comment:
Not sure this is the best method to translate a long text content. Maybe having HTML is more flexible. Order of sentences can vary in different languages.